cypress/integration/anton.sample/todoSample.feature

```diff
@@ -0,0 +1,51 @@
+Feature: Managing of todo list
+  As a user
+  I want to use todo list functionality
+  So I can plan my activities
+
+  Background:
+    When I add items via localStorage
```

Comment:
There shouldn't be any `When` or `Then` steps in Background section. Only `Given`.
Single purpose of background section - is to make nessecary preparations to start actual scenario. There is no point to validate items added in localStorage appear in todo list - because this is not scenario that could happen with real user. You just add them to be able to next, real action. If they aren't adding correctly, your next tests will fail anyway.